modules/phase_field/include/postprocessors/FeatureFloodCount.h

```diff
@@ -377,8 +377,8 @@ FeatureFloodCount::writeCSVFile(const std::string file_name, const std::vector<T
 {
   if (processor_id() == 0)
   {
-    // typdef makes subsequent code easier to read...
-    typedef std::map<std::string, MooseSharedPointer<std::ofstream> >::iterator iterator_t;
+    // alias declaration
+    using iterator_t = std::map<std::string, std::unique_ptr<std::ofstream> >::iterator;
```

Comment:
Hodor.

On Sun, May 29, 2016 at 11:50 PM, Daniel Schwen <notifications@github.com>
wrote:

> In modules/phase_field/include/postprocessors/FeatureFloodCount.h
> <https://github.com/idaholab/moose/pull/7106#discussion_r65021289>:
>
> > @@ -377,8 +377,8 @@ FeatureFloodCount::writeCSVFile(const std::string file_name, const std::vector<T
> >  {
> >    if (processor_id() == 0)
> >    {
> > -    // typdef makes subsequent code easier to read...
> > -    typedef std::map<std::string, MooseSharedPointer<std::ofstream> >::iterator iterator_t;
> > +    // alias declaration
> > +    using iterator_t = std::map<std::string, std::unique_ptr<std::ofstream> >::iterator;
>
> Uuuuuaaaaarrrrggggghhhhhhhhhb
> … <#m_6193524136164944466_>
> On Sun, May 29, 2016, 9:48 PM Derek Gaston ***@***.***> wrote: In
> modules/phase_field/include/postprocessors/FeatureFloodCount.h <#7106
> (comment)
> <https://github.com/idaholab/moose/pull/7106#discussion_r65021219>>: > @@
> -377,8 +377,8 @@ FeatureFloodCount::writeCSVFile(const std::string
> file_name, const std::vector<T > { > if (processor_id() == 0) > { > - //
> typdef makes subsequent code easier to read... > - typedef
> std::map<std::string, MooseSharedPointer<std::ofstream> >::iterator
> iterator_t; > + // alias declaration > + using iterator_t =
> std::map<std::string, std::unique_ptr<std::ofstream> >::iterator; Hah - it
> really is too bad that they had to kill Jon Snow again... — You are
> receiving this because you were mentioned. Reply to this email directly,
> view it on GitHub <
> https://github.com/idaholab/moose/pull/7106/files/ad920f1dbcbec4297d7a5b44f10dde1f559b44ee#r65021219>,
> or mute the thread <
> https://github.com/notifications/unsubscribe/AAMWPvPobht6IHDSLfFeL7dod0zJtu_sks5qGl4OgaJpZM4Io9-J>
> .
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/7106/files/ad920f1dbcbec4297d7a5b44f10dde1f559b44ee#r65021289>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHPulMLjbM4Iyy5ZCS-jj-884TeT0v3Lks5qGl6TgaJpZM4Io9-J>
> .
>